docs/_tutorials/getting-started.md

```diff
@@ -66,32 +67,31 @@ for step, batch in enumerate(data_loader):
     model_engine.step()
 ```
 
-
 Under the hood, DeepSpeed automatically performs the necessary operations
 required for distributed data parallel training, in mixed precision, with a
-pre-defined learning rate schedule:
+pre-defined learning rate scheduler:
 
-* **Gradient Averaging**: in distributed data parallel training, `backward`
+- **Gradient Averaging**: in distributed data parallel training, `backward`
   ensures that gradients are averaged across data parallel processes after
   training on an `train_batch_size`.
 
-* **Loss Scaling**: in FP16/mixed precision training, the DeepSpeed
+- **Loss Scaling**: in FP16/mixed precision training, the DeepSpeed
   engine automatically handles scaling the loss to avoid precision loss in the
   gradients.
 
-* **Learning Rate Schedule**: if using DeepSpeed's learning rate
-  schedule, then DeepSpeed automatically handles any updates to the learning
-  rate when `step` is executed.
-
-
+- **Learning Rate Scheduler**: when using a DeepSpeed's learning rate scheduler (specified in the `ds_config.json` file), DeepSpeed automatically updates the learning rate at every training step (when `model_engine.step()` is executed). When not using a DeepSpeed's learning rate scheduler:
+  - if the scheduler is supposed to be updated every training step, then the user can pass the scheduler to `deepspeed.initialize` when initializing the DeepSpeed engine and let DeepSpeed manage it for update or save/restore.
+  - if the scheduler is supposed to be updated every training epoch, then the user should NOT pass the scheduler to DeepSpeed during initialization and must manage it explicitly.
```

Comment:
This case should cover not just update at training epoch intervals, but any arbitrary intervals different from training step. So can we rewrite like, "if the schedule is supposed to be updated at any other interval (e.g., training epochs), ..."?

Reply:
Addressed.